static/js/compose_ui.js

```diff
@@ -144,6 +144,18 @@ export function wrap_text_with_markdown(textarea, prefix, suffix) {
     }
 }
 
+export function set_compose_box_top(set_top) {
+    if (set_top) {
+        const compose_top =
+            $("#navbar_alerts_wrapper").height() +
+            $(".header").height() +
+            Number.parseInt($(".header").css("paddingBottom"), 10);
+        $("#compose").css("top", compose_top + "px");
+    } else {
+        $("#compose").css("top", "");
+    }
+}
```

Comment:
I assume we now need to call this function whenever doing a show/hide in `navbar_alerts`.

Reply:
I think instead of calling this function every time `navbar_alerts` is shown or hidden, we should call this every time `.app` is resized, as the height of the expanded compose-box is closely related to the height of `.app`, and this will also consider the case when the window is resized or app in zoomed in/out.

Reply:
Updated the code.